app/tests/e2e/test_revocation.py

```diff
@@ -253,3 +260,211 @@ async def test_publish_revocations_bad_payload(
         )
 
     assert_that(exc.value.status_code).is_equal_to(404)
+
+
+@pytest.mark.anyio
+async def test_proof_revoked_credential_v1(
+    faber_client: RichAsyncClient,
+    issue_revocable_credentials_to_alice_and_revoke: list,  # pylint: disable=unused-argument
+    acme_client: RichAsyncClient,
+    acme_verifier: CreateTenantResponse,
+    alice_member_client: RichAsyncClient,
+    alice_tenant: CreateTenantResponse,
+    acme_and_alice_connection: AcmeAliceConnect,
+):
+
+    alice_proofs_listener = SseListener(
+        topic="proofs", wallet_id=alice_tenant.wallet_id
+    )
+    acme_proofs_listener = SseListener(
+        topic="proofs", wallet_id=acme_verifier.wallet_id
+    )
+
+    # Publish revoked credentials
+    await faber_client.post(
+        f"{CREDENTIALS_BASE_PATH}/publish-revocations",
+        json={"revocation_registry_credential_map": {}},
+    )
```

Comment:
I notice that this fixture:
`issue_revocable_credentials_to_alice_and_revoke`
still requires revocations to be published ... and that can get cumbersome if lots of tests need to do this, so I think we should have a fixture that includes the publishing. Some tests require the unpublished ones (for testing clear revocations), so let's just rename this one to something like:
`issue_revocable_creds_to_alice_and_revoke_unpublished`
and then make a new fixture:
`issue_revocable_creds_to_alice_and_revoke_published` which extends the unpublished one and uses faber to publish it.